example/Swift/PrebidDemo/PrebidDemo/BannerController.swift

```diff
@@ -108,6 +108,9 @@ class BannerController: UIViewController, GADBannerViewDelegate, MPAdViewDelegat
 
     func adViewDidReceiveAd(_ bannerView: GADBannerView) {
         print("adViewDidReceiveAd")
+        
+        self.dfpBanner.resize(bannerView.adSize)
```

Comment:
This needs to be documented here `http://prebid.org/prebid-mobile/pbm-api/ios/pbm-bannerad-ios.html` to include adManager and the implementation logics like these to address specific mediation partner issues